src/devices_models/devices/electric_loads.jl

```diff
@@ -248,7 +249,7 @@ function nodal_expression!(
         include_parameters(
             psi_container,
             ts_data_reactive,
-            UpdateRef{L}(REACTIVE_POWER, "get_maxactivepower"),
+            UpdateRef{L}(REACTIVE_POWER, "get_maxreactivepower"),
```

Comment:
we need to update the conditional here for both active and reactive power (this is the example of reactive. 

Case 1: Parameters and Forecast `UpdateRef{L}(REACTIVE_POWER, "get_maxreactivepower")`
Case 2: Parameters and No - Forecast `UpdateRef{L}(REACTIVE_POWER, "get_reactivepower")`

Reply:
what about case3: no paramaters, no forecast?